### PR TITLE
LSE Omnibus embedding

### LINKED ISSUE #550: Add LSE-OMNI
Should be able to run OMNI where each individual matrix that is input is the Laplacian of each.

Theory/papers on it have not been published (to my knowledge) but has definitely been used successfully in the wild.

Maybe we want to include R-LSE as an option?

### PR DESCRIPTION
**Pull request by Nyecarr**
- [ ] Does this PR add any new dependencies?
- [x] Does this PR modify any existing APIs?
   - [x] Is the change to the API backwards compatible?
- [x] Have you built the documentation (reference and/or tutorial) and verified the generated documentation is appropriate?

#### What does this implement/fix? Briefly explain your changes.
Add ability to create LSE omnibus embeddings. 

#### Any other comments?
Based off of @zeou1 PR

### CODE REVIEW

File: graspologic/embed/omni.py [hunk 1]
```diff
@@ -145,6 +168,10 @@ class OmnibusEmbed(BaseEmbedMulti):
         Only applicable for ``algorithm="randomized"``; allows you to seed the
         randomized svd solver for deterministic, albeit pseudo-randomized behavior.
 
+    lse : bool, optional (default False)
```
Comment (Nyecarr) [inline graspologic/embed/omni.py:171]: I pulled this from the earlier LSE Omni PR -- Is this the api we can all agree on or does someone have other suggestions?

In topologic, I believe we used an enum here (ase or lse, default ase.) Other devs may prefer a magic string but I'd prefer not. 

We can also leave this as is and change it in the future if it turns out we want to support something other than ase/lse.

File: tests/embed/test_omni.py [hunk 2]
```diff
@@ -213,3 +212,65 @@ def run(diag_aug):
...
+            return (arr[:n] + arr[n:]) / 2
+
+        def run(diag_aug):
+            X, A1, A2 = generate_data(1000, seed=2)
+            L1 = to_laplacian(A1)
+            L2 = to_laplacian(A2)
+            Lbar = (L1 + L2) / 2
+
+            omni = OmnibusEmbed(n_components=3, diag_aug=diag_aug, lse=True)
+            OmniBar = compute_bar(omni.fit_transform([csr_matrix(L1), csr_matrix(L2)]))
```
Comment (Nyecarr) [inline tests/embed/test_omni.py:265]: Works with sparse.

File: graspologic/embed/omni.py [hunk 3]
```diff
@@ -137,14 +160,18 @@ class OmnibusEmbed(BaseEmbedMulti):
...
         If graph(s) are directed, whether to concatenate each graph's left and right (out and in) latent positions
         along axis 1.
 
-    svd_seed : int or None (default ``None``)
+    svd_seed : int or None (default = ``None``)
         Only applicable for ``algorithm="randomized"``; allows you to seed the
         randomized svd solver for deterministic, albeit pseudo-randomized behavior.
 
+    lse : bool, optional (default = False)
+        Whether to construct the OMNI matrix use the laplacian matrices
```
Comment (daxpryce) [inline graspologic/embed/omni.py:172]: personally, I'd say we should say `Omni` instead of `OMNI`

**Comment by bdpedigo** [comment]
implementation looks fine, my continued queasyness is just that I don't have a simulation / data example that I can show someone that says when LSE omni would be preferred

**Comment by daxpryce** [comment]
> implementation looks fine, my continued queasyness is just that I don't have a simulation / data example that I can show someone that says when LSE omni would be preferred

what level of queasiness are we talking about?  like, we're currently using it to measure fluidity in our viva insights product to calculate https://docs.microsoft.com/en-us/workplace-analytics/azure-templates/ona-metric-calculations#fluidity

or are you more wondering if there's an academic paper on it?  because I don't think we have that

File: graspologic/embed/omni.py (hunk 3 above)
Comment (Nyecarr) [reply graspologic/embed/omni.py]: Updated

**Review by daxpryce — APPROVED** [review]

**Comment by bdpedigo** [comment]
> what level of queasiness are we talking about? 

not extremely, just think it would be good to have a public-facing reason for using this stuff as there is no associated publication, as you mention

**Comment by daxpryce** [comment]
should @nyecarr reference that page in the docs as well?
